### PR TITLE
[Feature:163751483]landing page

### PR DESCRIPTION
**Pull request by XROLE**
#### What does this PR do?
- Create LearnGround landing page

#### Description of Task to be completed?
- create landing page component 
- add style 
- integrate fontawesome cdn 
- install materialize-css 
- install google font configure 
- webpack to use google font

#### How should this be manually tested?
- run `npm install ` and `npm start:dev` to see the landing page

#### What are the relevant pivotal tracker stories?
- [163751483](https://www.pivotaltracker.com/story/show/163751483)

#### Screenshots (if appropriate)
screen shot 2019-02-07 at 1 11 13 pm

### CODE REVIEW

**Comment by andela-mabdussalam** [comment]
@XROLE coverage decreased. Look into it.

**Comment by XROLE** [comment]
> @XROLE coverage decreased. Look into it.

Fixing it asap

**Comment by XROLE** [comment]
@andela-mabdussalam I've effected the feedback. Coverage increased

File: src/app/home/home.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
+body {
+    margin: 0;
+    padding: 0;
+}
+
+#landingPage {
+    height: 85%;
```
Comment (rajeman) [inline src/app/home/home.css]: why do you have both css and scss files?

File: src/app/home/home.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    width: auto;
+    background-image: url('../../img/landingPageImage.jpeg');
+    background-size: cover;
+    position: relative;
+    background-position: center center;
+    color: white;
+}
+
```
Comment (rajeman) [inline src/app/home/home.css]: Why is this file not in the style folder in /src

Comment (XROLE) [reply src/app/home/home.css]: moving it asap

File: src/app/home/home.css (hunk 1 above)
Comment (XROLE) [reply src/app/home/home.css]: Where exactly? I only added home.css to style the homepage component

**Comment by andela-mabdussalam** [comment]
Please update PR or add wip label.

File: src/app/home/HomeComponent.jsx [hunk 3]
```diff
@@ -1,11 +1,134 @@
...
+            <div className="get-started">
+              <p> Get Started </p>
+            </div>
+          </div>
+        </div>
       </div>
-    );
-  }
+      <div className="about-div">
+        <div style={{ width: '65%', margin: '0 auto' }}>
```
Comment (andela-mabdussalam) [inline src/app/home/HomeComponent.jsx]: @XROLE inline style.

Comment (XROLE) [reply src/app/home/HomeComponent.jsx]: Missed out this one. I removing it now.

File: src/tests/setup.js [hunk 4]
```diff
@@ -1,5 +0,0 @@
-/* eslint-disable import/no-extraneous-dependencies */
```
Comment (andela-mabdussalam) [inline src/tests/setup.js:1]: @XROLE Why was this file deleted?

Comment (XROLE) [reply src/tests/setup.js:1]: The file is now in the root directory. Since test directory has been removed.

**Review by andela-mabdussalam — APPROVED** [review]
